### ISSUE TITLE
Info placeholders for empty datasets

### ISSUE THREAD

**Issue by cadavre** [metasfresh/metasfresh-webui-frontend#12]

**Comment by cadavre** [comment]
How we can get empty placeholder info from API? @teosarca how you gonna provide it per tab?

**Comment by teosarca** [comment]
For empty fields, i've introduced "emptyText" to field layout.
e.g.

```
"elements": [
                    {
                      "caption": "M_Warehouse_ID",
                      "description": "Lager oder Ort für Dienstleistung",
                      "widgetType": "List",
                      "type": "primary",
                      "fields": [
                        {
                          "field": "M_Warehouse_ID",
                          "source": "list",
                          "emptyText": "nicht"
                        }
                      ]
                    }
                  ]
```

See https://github.com/metasfresh/metasfresh-webui/issues/5

**Comment by cadavre** [comment]
Nice.

Now only one more: empty placeholder per tab table. We have it for current iteration, so ping us @teosarca when ready.

**Comment by teosarca** [comment]
For empty tabs, see https://github.com/metasfresh/metasfresh-webui/issues/6

**Comment by damianprzygodzki** [comment]
Resolved by 235468d7c3d1c5374f92f240445a9f3f8cf3f864
